plugins/content_types/search_result.inc

```diff
@@ -83,10 +83,22 @@ function ting_search_search_result_content_type_render($subtype, $conf, $panel_a
 
     // Set message.
     $message = '';
+
+    $frequency_rank = array(
+      'rank_main_title' => t('Title'),
+      'rank_creator' => t('Creator'),
+      'rank_subject' => t('Subject')
+    );
+    if (isset($search_result) && in_array($search_result->sortUsed, array_keys($frequency_rank))) {
+      $msg = t('Records are sorted by %sortUsed . Select another sort if it is not suited', array('%sortUsed' => $frequency_rank[$search_result->sortUsed]));
+      $message .= '<div class="messages status">' . $msg . '</div>';
```

Comment:
But i would like the two messages to be shown with different styling eg. :
this one as a warning (yellow) :
"Your search gave more than 100 results. Try to search more specific or use the facets to filter the result."

and this one as a status (green):
"Records are sorted by Title . Select another sort if it is not suited"